domains/dhananjaypatil.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dhananjay8",
+        "email": "dhananjay.pict@gmail.com"
+    },
+    "record": {
+        "CNAME": "dhananjay8.github.io"
```

Comment:
site can't be reached.

Reply:
Sorry for inconvenience. Please check again. I have hosted it again. Thanks.